### ISSUE TITLE
Page issue: Tablet setup in Krita

### ISSUE THREAD

**Issue by warownia1** [DIGImend/digimend.github.io#21]
There is an issue with page [Tablet setup in Krita](http://digimend.github.io/support/howto/apps/krita/):

The current version of Krita (4.3.0) is using Qt5. Also, the mentioned bug about Qt4.8 applications not detecting pen pressure pressure is already closed. Please verify whether the lack of pressure sensitivity is still an issue and update the website.

**Comment by spbnick** [comment]
Yeah, the HOWTOs are really old, I'd be happy if somebody contributed an update!
